### PR TITLE
Changelog: add extra entries to release 0.4.1

### PR DESCRIPTION
**Pull request by deeplow**
Adds some extra changelog entries that we missed since our last 0.4.1 release candidate.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,10 +24,12 @@ since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.or
 - Development: Add dummy isolation provider for testing non-conversion-related
   issues in virtualized Windows and MacOS, where Docker can't run, due to the
   lack of nested virtualization ([issue #229](https://github.com/freedomofpress/dangerzone/issues/229))
-- Add support for more MIME types that were previously disregarded ([issue #377](https://github.com/freedomofpress/dangerzone/issues/377))
+- Add support for more MIME types that were previously disregarded ([issue #369](https://github.com/freedomofpress/dangerzone/issues/377))
```
Comment (sudwhiwdh) [inline CHANGELOG.md:27]: ([issue #369](https://github.com/freedomofpress/dangerzone/issues/369))?

[the remark this answers is not in the document]
Comment (sudwhiwdh) [reply CHANGELOG.md]: Thank you for your work on Dangerzone!

[the remark this answers is not in the document]
Comment (deeplow) [reply CHANGELOG.md]: You're welcome! :slightly_smiling_face:

**Comment by apyrgio** [comment]
I took a look at the commits we have pushed since then and I think you're not missing something. Ideally, I'd merge the PR as is, but I cannot squash your fixup without invalidating the other commit signatures. So, I'll sign them as myself, so that @eloquence can merge them.

**Comment by apyrgio** [comment]
Pushed one more fix in this PR, because we pointed to the wrong issue (#219 vs #50) for the MacOS Apple Silicon platform support.

**Review by eloquence — APPROVED** [review]

**Pull request merged**